includes/libraries/action-scheduler/classes/ActionScheduler_Abstract_ListTable.php

```diff
@@ -107,7 +107,7 @@ abstract class ActionScheduler_Abstract_ListTable extends WP_List_Table {
 	 * `_x` with some default (the package name)
 	 */
 	protected function translate( $text, $context = '' ) {
-		return _x( $text, $context, $this->package );
+		return $text;
```

Comment:
Could you please clarify this change? What is the point of returning `$text` without any modification? Maybe the idea is to deprecate this function? If so, should we update the docblock?